code/mod_table/mod_table.h

```diff
@@ -42,6 +42,7 @@ extern bool Cutscene_camera_displays_hud;
 extern bool Alternate_chaining_behavior;
 extern bool Fixed_chaining_to_repeat;
 extern bool Use_host_orientation_for_set_camera_facing;
+extern bool Use_model_eyepoint_for_set_camera_facing;
```

Comment:
Since the issue you are addressing is to use the model eyepoint for `set-camera-host`, both the variable and the text parsed in game_settings.tbl should refer to set-camera-host, not set-camera-facing.